src/ui/display.rs

```diff
@@ -176,3 +186,28 @@ impl OriginDimensions for DisplayExclusive {
         Size::new(self.0.var_screen_info.xres, self.0.var_screen_info.yres)
     }
 }
+
+impl<'a> DrawTarget for DisplayRotated<'a> {
+    type Color = BinaryColor;
+    type Error = core::convert::Infallible;
+
+    fn draw_iter<I>(&mut self, pixels: I) -> Result<(), Self::Error>
+    where
+        I: IntoIterator<Item = Pixel<Self::Color>>,
+    {
+        let res = self.size();
```

Comment:
Add a comment about this only doing one kind of rotation (the one we need for the button legend).